sys/libkern/gsb_crc32.c

```diff
@@ -747,7 +747,7 @@ multitable_crc32c(uint32_t crc32c,
 	if (length == 0) {
 		return (crc32c);
 	}
-	to_even_word = (4 - (((vaddr_t) buffer) & 0x3));
+	to_even_word = (4 - (((ptraddr_t) buffer) & 0x3));
```

Comment:
Should work with upstream's uintptr_t

Reply:
Yeah that is almost certainly an offset interpretation change...